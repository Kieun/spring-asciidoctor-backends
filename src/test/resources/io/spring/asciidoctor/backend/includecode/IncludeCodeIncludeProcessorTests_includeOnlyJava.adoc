= Test Document
:docs-java: {docdir}/javacode

[[some.convention-based-anchor]]
== Some Convention Based Anchor

include::code:SomeCode[]
