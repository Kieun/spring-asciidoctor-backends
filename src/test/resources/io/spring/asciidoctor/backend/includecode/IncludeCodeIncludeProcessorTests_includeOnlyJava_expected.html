<div class="listingblock">
 <div class="content">
  <pre class="highlight"><code class="language-java" data-lang="java">somejava
</code></pre>
 </div>
</div>